docs/develop/tools-and-features/gsoc.md

```diff
@@ -0,0 +1,322 @@
+---
+title: GSOC
+id: gsoc
+---
+
+## Introduction
+
+The Graffiti Single Owner Chunk (GSOC) feature enables a single Bee *service* node to receive messages from multiple Bee *writer* nodes. It is based on a [Single Owner Chunk (SOC)](/docs/develop/tools-and-features/chunk-types/#single-owner-chunks), but unlike a standard SOC, which can be updated only by a single owner, GSOC allows multiple nodes to make updates. The GSOC address is derived so that it falls within the neighborhood of the service node, ensuring updates are automatically synced as part of the normal full node syncing process.  
```

Comment:
> unlike a standard SOC, which can be updated only by a single owner, GSOC allows multiple nodes to make updates

I am not sure about this part. It IS a standard SOC. The only difference is that the signer (PK) is known by multiple parties, and it is deliberately mined to fall into a specific neighborhood.